pypsa-to-iamc

```diff
@@ -0,0 +1,634 @@
+            var[sh]['Final energy consumption|Transportation|Road|Liquids|Fossil'] = 0.001*ct*(n.loads_t.p.filter(like ='land transport oil').filter(like =country).sum().sum())                                                               
+            var[sh]['Final energy consumption|Transportation|Liquids|Fossil'] = (0.001*ct*(n.loads_t.p.filter(like ='shipping oil').filter(like =country).sum().sum())
+                  +var[sh]['Final energy consumption|Transportation|Road|Liquids|Fossil'])
+
+            var[sh]['Final energy consumption|Transportation|Electricity'] = 0.001*ct*(n.loads_t.p.filter(like ='land transport EV').filter(like =country).sum().sum())                                                               
+                                                                               
+            var[sh]['Final energy consumption|Electricity'] = (0.001*ct*(n.loads_t.p[[i for i in n.loads.index if i==country+'0 0']].sum().sum())
+                  +var[sh]['Final energy consumption|Transportation|Electricity']+var[sh]['Final energy consumption|Industries|Electricity'])
+            var[sh]['Final energy consumption|Gases|Hydrogen'] = (var[sh]['Final energy consumption|Industries|Gases|Hydrogen']
+                  +var[sh]['Final energy consumption|Transportation|Gases|Hydrogen'])
+
+                                                                               
+            var[sh]['Final energy consumption|Buildings|Heating|District heating'] = 0.001*ct*(n.loads_t.p.filter(like ='urban central heat').filter(like =country).sum().sum())   
+            var[sh]['Final energy consumption|District heating']=var[sh]['Final energy consumption|Buildings|Heating|District heating']                                                       
+            var[sh]['Final energy consumption|Buildings|Heating'] = 0.001*ct*(n.loads_t.p.filter(like ='residential rural heat').filter(like =country).sum().sum()
+                  +n.loads_t.p.filter(like ='services rural heat').filter(like =country).sum().sum()+n.loads_t.p.filter(like ='residential urban decentral heat').filter(like =country).sum().sum()                                              
+                  +n.loads_t.p.filter(like ='services urban decentral heat').filter(like =country).sum().sum()+ var[sh]['Final energy consumption|District heating'] )                                 
+                                                                               
+                 #add residential/services water/space heating demand if we have them                                                   
+                                                                               
+            industry_demand=pd.read_table('resources/industrial_energy_demand_elec_s370_37m_{}.csv'.format(year),delimiter=',',index_col=0)
+            var[sh]['Final energy consumption|Industries|Coal|Coal products'] = 0.001*ct*industry_demand['coke'].filter(like=country).sum()*(10e5/3)
```

Comment:
define constant 10e5/3 also at the beginning

Reply:
I actually realized it doesn't seem to need conversion and deleted it altogether.